indico/modules/events/notifications.py

```diff
@@ -27,14 +42,46 @@ def notify_event_creation(event, occurrences=None):
              filter(Category.notify_managers | (Category.event_creation_notification_emails != []))
              .options(joinedload('acl_entries')))
     for cat in query:
-        emails.update(cat.event_creation_notification_emails)
-        if cat.notify_managers:
-            for manager in cat.get_manager_list():
-                if manager.principal_type in (PrincipalType.user, PrincipalType.email):
-                    emails.add(manager.email)
-                elif manager.principal_type in (PrincipalType.local_group, PrincipalType.multipass_group):
-                    emails.update(x.email for x in manager.get_members())
+        emails.update(_get_emails_from_category(cat))
 
     if emails:
         template = get_template_module('events/emails/event_creation.txt', event=event, occurrences=occurrences)
         send_email(make_email(bcc_list=emails, template=template))
+
+
+def notify_move_request_creation(events, target_category, comment=''):
+    """Send email notifications when a move request is created.
+
+    :param events: List of events requested to move.
+    :param target_category: The category to move the events into.
+    :param comment: Optional requestor's comment.
+    """
+    emails = set()
+    query = (target_category.chain_query.
```

Comment:
that `.` should be in the next line before `filter` (please don't fix it in `notify_event_creation` above, the unlisted events PR already does that)